tests/foreman/ui/test_host.py

```diff
@@ -1,14 +1,278 @@
+        4. Creates new content-view by associating rhel67 repository
+        5. Publish and promote the content-view to next environment
+        6. Search for puppet environment and associate location
+        7. Search for smart-proxy and associate location
+        8. Search for domain and associate org, location and dns proxy
+        9. Search for '192.168.100.0' network and associate org, location,
+        dns/dhcp/tftp proxy, and if its not there then creates new
+        10. Creates new compute-resource
+        11. Search 'Kickstart default' partition table and rhel67 OS along with
+        provisioning/PXE templates
+        12. Associates org, location and OS with provisioning and PXE templates
+        13. Search for x86_64 architecture
+        14. Associate arch, partition table, provisioning/PXE templates with OS
+        15. Search for media and associate org/location
+        16. Create new host group with all required entities
+
+        """
+        # Create a new Organization and Location
+        cls.org = entities.Organization(name=gen_string('alpha')).create()
+        cls.org_name = cls.org.name
+        cls.loc = entities.Location(
+            name=gen_string('alpha'),
+            organization=[cls.org]
+        ).create()
+        cls.loc_name = cls.loc.name
+
+        # Create a new Life-Cycle environment
+        cls.lc_env = entities.LifecycleEnvironment(
+            name=gen_string('alpha'),
+            organization=cls.org
+        ).create()
+
+        # Create a Product, Repository for custom RHEL6 contents
+        cls.product = entities.Product(
+            name=gen_string('alpha'),
+            organization=cls.org).create()
+        cls.repo = entities.Repository(
+            name=gen_string('alpha'),
+            product=cls.product,
+            url=conf.properties['clients.rhel6_repo']
+        ).create()
+
+        # Increased timeout value for repo sync
+        entity_mixins.TASK_TIMEOUT = 3600
+        cls.repo.sync()
+
+        # Create, Publish and promote CV
+        cls.content_view = entities.ContentView(
+            name=gen_string('alpha'),
+            organization=cls.org).create()
+        cls.content_view.repository = [cls.repo]
+        cls.content_view = cls.content_view.update(['repository'])
+        cls.content_view.publish()
+        cls.content_view = cls.content_view.read()
+        promote(cls.content_view.version[0], cls.lc_env.id)
+
+        # Search for puppet environment and associate location
+        cls.environment = entities.Environment().search(
+            query={'organization_id': [cls.org.id]}
+        )[0]
+        cls.environment.location = [cls.loc]
+        cls.environment = cls.environment.update(['location'])
+
+        # Search for SmartProxy, and associate location
+        cls.proxy = entities.SmartProxy().search(
+            query={
+                u'search': u'name={0}'.format(
+                    conf.properties['main.server.hostname'])
```

Comment:
Moving the parentheses to next line will make it more consistent with other code:
```py
        cls.proxy = entities.SmartProxy().search(
            query={
                u'search': u'name={0}'.format(
                    conf.properties['main.server.hostname']
                )
            }
        )[0]
```